compiler/llvm/llvmGlobalToWide.cpp

```diff
@@ -2209,7 +2209,17 @@ Type* createWidePointerToType(Module* module, GlobalToWideInfo* i, Type* eltTy)
   // Get the wide pointer struct containing {locale, address}
   Type* fields[2];
   fields[0] = i->localeIdType;
-  fields[1] = PointerType::get(eltTy, 0);
+  llvm::PointerType* ptrTy;
+  if (eltTy) {
```

Comment:
Can you add a comment to this function saying `eltType` can be `nullptr` to indicate an opaque pointer type?